package.json

```diff
@@ -60,5 +61,8 @@
     "files": [
       "test/tests/*"
     ]
+  },
+  "dependencies": {
+    "npm-merge-driver": "^2.3.5"
```

Comment:
Should this be under "devDependencies" too?

Reply:
Maybe it should 🙂 Can you file an issue about this, and cc myself and @spectranaut ?